storage/src/vespa/storage/common/storagelink.cpp

```diff
@@ -165,9 +165,9 @@ void StorageLink::sendUp(const shared_ptr<StorageMessage> & msg)
         default:
             LOG(error, "Link %s trying to send %s up while in state %s",
                 toString().c_str(), msg->toString(true).c_str(), stateToString(getState()));
-            assert(false);
+            return;
```

Comment:
There was one test that tested enque/deque ordering that triggered this. It posted N tasks BEFORE doing open(), and then checking that they were received in same order as posted.
With a queue it would wait in the queue until open was called. That would now trigger the assert as it is open once constructed. So I concluded the assert could just be removed.